ui/pages/settings/developer-options-tab/developer-options-tab.tsx

```diff
@@ -209,11 +210,53 @@ const DeveloperOptionsTab = () => {
     );
   };
 
+  const renderRemoteFeatureFlags = () => {
+    const remoteFeatureFlags = useSelector(getRemoteFeatureFlags);
+    return (
+      <Box
+        className="settings-page__content-row"
+        display={Display.Flex}
+        flexDirection={FlexDirection.Row}
+        justifyContent={JustifyContent.spaceBetween}
+        gap={4}
+      >
+        <div className="settings-page__content-item">
+          <span>Remote feature flags</span>
+          <div className="settings-page__content-description">
+            Remote feature flag values come from the LaunchDarkly by default. If
+            you need to update feature flag values locally for development
+            purposes, you can change feature flag values in
+            .manifest-overrides.json, which will override values coming from
+            LaunchDarkly.
```

Comment:
nit (it is fine as is, but could be improved by removing the `the` before `LaunchDarkly`):

```suggestion
            Remote feature flag values come from LaunchDarkly by default. If
            you need to update feature flag values locally for development
            purposes, you can change feature flag values in
            .manifest-overrides.json, which will override values coming from
            LaunchDarkly.
```

Reply:
Sure ! modified in 9f92cc95319d6276c4b26b0f61ef7a96eb890fc1